examples/tracing-http-propagator/src/server.rs

```diff
@@ -30,11 +41,11 @@ async fn handle_health_check(
     _req: Request<Incoming>,
 ) -> Result<Response<BoxBody<Bytes, hyper::Error>>, Infallible> {
     let tracer = get_tracer();
-    let mut span = tracer
+    let _span = tracer
```

Comment:
I think it would be worth calling out why you haven't assigned this to `_` (presumably because you don't want it to be dropped until the end of the function?)

Reply:
`_` is fine here. modified.